# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../hostap-driver/nodepmod.patch
# Copyright (C) 2004 - 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- hostap-driver-0.4.7/Makefile.orig	2005-03-19 05:17:28.000000000 +0100
+++ hostap-driver-0.4.7/Makefile	2006-05-04 10:27:33.000000000 +0200
@@ -98,6 +99,7 @@
 endif
 
 2.4:
+	echo "Kernelver: $(KERNELVER) Kernelrelease: $(KERNELRELEASE) Modpath: $(MODPATH)"
 	@if ! echo "$(KERNELRELEASE)" | grep -q "^2\.4"; then \
 		echo "*** Can't build for 2.4 with a non-2.4 source!"; \
 		exit 1; \
@@ -106,6 +108,7 @@
 	$(MAKE) install_all_info
 
 2.6:
+	echo "Kernelver: $(KERNELVER) Kernelrelease: $(KERNELRELEASE) Modpath: $(MODPATH)"
 	@if ! echo "$(KERNELRELEASE)" | grep -q "^2\.6" && \
 	    ! echo "$(KERNELRELEASE)" | grep -q "^2\.5"; then \
 		echo "*** Can't build for 2.6 with a non-2.6 source!"; \
@@ -150,12 +153,7 @@
 	@if [ -r $(MODPATH)/kernel/drivers/net/wireless/hostap_crypt.ko ]; then \
 		echo "You may need to remove old $(MODPATH)/kernel/drivers/net/wireless/hostap_crypt.ko"; \
 	fi
-	@if [ -f "/boot/System.map-$(KERNELRELEASE)" ] || [ "$(CURRENTKV)" = "$(KERNELRELEASE)" ]; then \
-		echo "*** Running depmod..."; \
-		/sbin/depmod -ae; \
-	else \
-		echo "*** I won't run depmod because I can't find an appropriate System.map file or you are not running the same kernel as the one you are building against. Please run it manually when the conditions are correct..."; \
-	fi
+	echo "T2: not running depmod in build."
 
 install_hostap: $(MSRC)/hostap.o
 	@echo "Installing hostap.o to $(MODPATH_PCI)"
@@ -175,7 +173,7 @@
 	mkdir -p $(MODPATH_CS)
 	cp -f $(MSRC)/$(MODULE_CS) $(MODPATH_CS)
 ifndef DESTDIR
-	/sbin/depmod -ae
+	echo "T2: not running depmod in build."
 endif
 
 install_conf:
@@ -191,13 +189,13 @@
 	@echo "Installing $(MODULE_PLX) to $(MODPATH_PLX)"
 	mkdir -p $(MODPATH_PLX)
 	cp -f $(MSRC)/$(MODULE_PLX) $(MODPATH_PLX)
-	/sbin/depmod -ae
+	echo "T2: not running depmod in build."
 
 install_pci: install_check install_crypt install_hostap
 	@echo "Installing $(MODULE_PCI) to $(MODPATH_PCI)"
 	mkdir -p $(MODPATH_PCI)
 	cp -f $(MSRC)/$(MODULE_PCI) $(MODPATH_PCI)
-	/sbin/depmod -ae
+	echo "T2: not running depmod in build."
 
 uninstall:
 	find $(MODPATH) -name "hostap*" | xargs rm -f
